README.md

```diff
@@ -1,7 +1,81 @@
 # Papyros
 
-Scratchpad for multiple programming languages in the browser. 
+<p align="center">
+  <a href="https://www.npmjs.com/package/@dodona/papyros">
+    <img src="https://img.shields.io/npm/v/@dodona/papyros.svg" alt="Version of the npm package">
+  </a>
+  <a href="https://github.com/dodona-edu/papyros/actions?query=branch%3Amain">
+    <img src="https://github.com/dodona-edu/papyros/actions/workflows/deploy-pages.yaml/badge.svg" alt="GitHub checks status">
+  </a>
+  <a href="https://github.com/dodona-edu/papyros/blob/main/LICENSE">
+    <img alt="Source code license" src="https://img.shields.io/github/license/dodona-edu/papyros">
+  </a>
+</p>
 
-Python support is powered by Pyodide. \
-JavaScript is supported natively in the browser.\
-Other languages are currently out of scope, but the application is designed to be able to be extended.\
+Papyros is a programming scratchpad in the browser. It allows running arbitrary
+in various programming languages directly in your browser, no installation required.
+By taking away obstacles between students and coding, the learning experience becomes
+smoother and less error-prone. 
+
+Papyros aims to be:
+
+- **Easy to use** by having minimal installation instructions and an intuitive user interface
+- **Flexible** to support many programming languages
```

Comment:
I would leave this out. Since we only support 2 languages, the "many" is not really correct. The installation instructions from the first point is also invalid since you don't have to install a thing.